tools/pipelines/templates/build-npm-package.yml

```diff
@@ -357,19 +357,20 @@ extends:
                           testCoverage: ${{ parameters.testCoverage }}
 
                   - ${{ if contains(convertToJson(parameters.taskTest), 'tinylicious') }}:
-                    # Copy files so all artifacts we publish end up under the same parent folder.
-                    # Source is wherever the tinylicious log ends up in, after executing tests above.
-                    # CopyFiles@2 failed with OOM for some reason so doing it "by hand".
                     - task: Bash@3
-                      displayName: Copy tinylicious.log to artifact staging directory
+                      displayName: Upload tinylicious log
                       condition: succeededOrFailed()
                       continueOnError: true # Keep running subsequent tasks even if this one fails (e.g. the tinylicious log wasn't there)
                       inputs:
                         targetType: inline
-                        workingDirectory: '${{ parameters.buildDirectory }}/packages/test/test-end-to-end-tests'
                         script: |
-                          mkdir -p $(Build.ArtifactStagingDirectory)/tinyliciousLog
-                          cp tinylicious.log $(Build.ArtifactStagingDirectory)/tinyliciousLog/tinylicious.log
+                          PATH_TO_TINYLICIOUS_LOG=$(Build.SourcesDirectory)/packages/test/test-end-to-end-tests/tinylicious.log;
+                          if [ -f $PATH_TO_TINYLICIOUS_LOG ] ; then
+                            echo "Found file at '$PATH_TO_TINYLICIOUS_LOG'. Uploading.";
+                            echo "##vso[task.uploadfile]$PATH_TO_TINYLICIOUS_LOG";
```

Comment:
nit: perhaps add a small comment with an explanation or pointing to docs would help future maintainers understand what's going on here.